Library/Homebrew/formula.rb

```diff
@@ -1063,6 +1063,18 @@ def std_cmake_args
     ]
   end
 
+  # Returns the path to a make 3.81 or newer binary.
+  # Returns the system make on Leopard and newer, and the
+  # path to the actually-installed make on Tiger or older.
+  def make_path
```

Comment:
Where will this method be called? Would it be better to have this make path logic been implemented in superenv?